eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/spout/SpoutOutputCollectorWrapper.java

```diff
@@ -103,7 +103,7 @@ public SpoutOutputCollectorWrapper(CorrelationSpout spout,
          */
         List<Object> convertedTuple = converter.convert(tuple);
         if (convertedTuple == null) {
-            LOG.warn("source data {} can't be converted to a stream, ignore this message", tuple);
+            LOG.debug("source data {} can't be converted to a stream, ignore this message", tuple);
```

Comment:
If u are log the message without manipulating, it is unnecessary to add isDebugEnabled check since debug method will do the same thing.